### PR TITLE
Update cloud-cli-installation.md

### PR DESCRIPTION
**Pull request by greg-mckeon**
Added some more context for cloud cli install based on user feedback.

### CODE REVIEW

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 1]
```diff
@@ -24,17 +24,10 @@ brew install dbt-cloud-cli
 Verify the installation:
```
Comment (runleonarun) [inline website/docs/docs/cloud/cloud-cli-installation.md]: I moved this up so you read it before executing commands. @greg-mckeon , is `.zshrc` and bash Mac OS specific? Should we make this more OS agnostic or add something Windows specific?
```suggestion
Verify the installation by requesting your homebrew installation path (not your dbt core installs). If the `which dbt` command returns nothing, then you should modify your PATH in `~.zshrc` or create an alias.
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 2]
```diff
@@ -24,17 +24,10 @@ brew install dbt-cloud-cli
...
 
 ```bash
-which dbt
+which dbt  -- Should return your homebrew install path, not your dbt core installs. If not, you'll need to modify your PATH in ~.zshrc or create an alias.
```
Comment (runleonarun) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
which dbt
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 3]
```diff
@@ -89,14 +82,17 @@ You can find your project ID by selecting your project and clicking on **documen
 project-id: '123456'
 ```
 
-8. The following commands are supported as of now (we will be adding more in the future):
+You can find your project ID by selecting your project and clicking on **develop** in the navigation bar. Your project ID is the number in the URL: https://cloud.getdbt.com/develop/26228/projects/__{project id}__.
```
Comment (runleonarun) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
You can find your project ID by selecting your project and clicking on **Develop** in the navigation bar. Your project ID is the number in the URL: https://cloud.getdbt.com/develop/26228/projects/`PROJECT_ID`.
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 4]
```diff
@@ -89,14 +82,17 @@ You can find your project ID by selecting your project and clicking on **documen
...
 
-- `dbt run`
-- `dbt build`
-- `dbt deps`
-- `dbt cancel`
-- `dbt reattach`
-- `dbt --help`
+If dbt_cloud.yml already exists, edit it to ensure you have a project-id field with a valid project ID.
+
```
Comment (runleonarun) [inline website/docs/docs/cloud/cloud-cli-installation.md:86]:
```suggestion
```

File: website/docs/docs/cloud/cloud-cli-installation.md [hunk 5]
```diff
@@ -89,14 +82,17 @@ You can find your project ID by selecting your project and clicking on **documen
...
-8. The following commands are supported as of now (we will be adding more in the future):
+You can find your project ID by selecting your project and clicking on **develop** in the navigation bar. Your project ID is the number in the URL: https://cloud.getdbt.com/develop/26228/projects/__{project id}__.
 
-- `dbt run`
-- `dbt build`
-- `dbt deps`
-- `dbt cancel`
-- `dbt reattach`
-- `dbt --help`
+If dbt_cloud.yml already exists, edit it to ensure you have a project-id field with a valid project ID.
```
Comment (runleonarun) [inline website/docs/docs/cloud/cloud-cli-installation.md]:
```suggestion
If dbt_cloud.yml already exists, edit the file and verify the project ID field uses a valid project ID.
```

**Review by runleonarun — CHANGES_REQUESTED** [review]
Hey @greg-mckeon, I made some small suggestions and had just one question,

File: website/docs/docs/cloud/cloud-cli-installation.md (hunk 1 above)
Comment (greg-mckeon) [reply website/docs/docs/cloud/cloud-cli-installation.md]: this is mac or linux specific. there isn't really a more agnostic option, but most beta users are on mac. it seems ok to leave for now.

**Review by runleonarun — APPROVED** [review]
Looks good! Let's 🚢 it!

**Pull request merged**